Gruntfile.js

```diff
@@ -1,9 +1,11 @@
 /**
  * @author Adam Meadows [@job13er](https://github.com/job13er)
- * @copyright 2015 Cyan, Inc. All rights reserved.
+ * @copyright 2015 Ciena Corporation. All rights reserved.
 */
 
-var helper = require('./src/grunt/helper');
+'use strict';
```

Comment:
Did you intentionally bring back `use strict` or is it because the `.eslintrc` is wrong?